assets/css/elements/_layout.scss

```diff
@@ -1,16 +1,22 @@
 html {
   box-sizing: border-box;
+  height: 100%;
+  width: 100%;
+}
+
+body {
+  min-height: 100%;
+}
+
+main {
```

Comment:
It’d be great to move this to a class (like `app-main` or something) and into `components/`. `body` and `html` are required elements in any HTML document, but `main` is something we added and it’s contextual to the individual screens. So I like using classes in that case over far-reaching element styles. I have no idea if I just made any sense.

Perhaps an example would be useful: there are cases where two `main` elements are allowed. Having a dialog/modal is one…when the dialog is _shown_, it’s `main` is considered the primary content and the screen underneath should hide (`hidden`) it’s `main` element. So this is a case where we would probably not want these two `main` elements styled the same and having a class allows you to apply styles more contextually.

Reply:
Yes, that makes sense! :+1: